### DISCUSSION TITLE
Unread setup does not work with Yarr

### DISCUSSION THREAD

**Discussion by dpecos** [nkanaev/yarr#190] [Q&A]
Hi!

I'm trying to setup my new instance of yarr (docker image, sitting behind traefik, no passwd) with Unread, but the setup process just fails straight away with a 404.

I've tried the URL in my browser and I'm getting a 404 as well:

GET http://127.0.0.1:7070/fever?api

(this time running the same container from my local)

Am I doing anything wrong?

Thanks a lot!

Dani.

**Comment by dpecos** [comment]
To add more details, it's complaining about an incorrect MIME type (text/html vs JSON), but I think that's because of the 404.

**Comment by asimpson** [comment]
I ran into this too. The latest release (v2.4) doesn't have Unread fever support. You'll need to clone the repo and build it. Once you do that it should work fine. Hope this helps 🤞

**Reply by dpecos** [reply]
How do you enable support for fever once you clone the repo? After having a quick look into the server code, everything looks fine.

**Reply by asimpson** [reply]
Fever support is enabled by default.

1. Clone repo
2. Build
3. Run resulting binary 
4. `/fever` should be available then
